ext/oci8/oci8.stub.php

```diff
@@ -560,6 +560,7 @@ function oci_register_taf_callback($connection, ?callable $callback): bool {}
 /** @param resource $connection */
 function oci_unregister_taf_callback($connection): bool {}
 
+#[AllowDynamicProperties]
```

Comment:
```suggestion
#[\AllowDynamicProperties]
```

For consistency with https://github.com/php/php-src/commit/ff472ce6fc462bfeb6d88a651eec3779c0923dbe#diff-3e14bb712f3473bad87528bc5402c149d67ef958a219106622028b38b2848eccR5 and because Stubs do not appear to support `use …`, making the leading backslash required for namespaced classes.